app/src/main/res/values/styles.xml

```diff
@@ -1,11 +1,13 @@
 <resources>
-
-    <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.AppCompat.Light.DarkActionBar">
-        <!-- Customize your theme here. -->
         <item name="colorPrimary">@color/colorPrimary</item>
         <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
         <item name="colorAccent">@color/colorAccent</item>
     </style>
-
+    <style name="MainText" parent="TextAppearance.AppCompat">
```

Comment:
Since your styles only contain a single attribute you don't really gain that much by extracting them, are there any more attributes you could extract, maybe by having more styles?

Reply:
Your new styles look much more useful, nice work 🙂